packages/core/src/components/table/table/test/expandable-row-methods/expandable-row-methods.e2e.ts

```diff
@@ -0,0 +1,25 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/table/table/test/expandable-row-methods/index.html';
+
+test.describe.parallel('tds-table-expandable-row-methods', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(componentTestPath);
+    const tableComponent = page.getByRole('table');
+    await expect(tableComponent).toHaveCount(1);
+    await tableComponent.waitFor({ state: 'visible' });
+  });
+
+  test('click expand button', async ({ page }) => {
+    page.locator('#expand-button').click();
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0.05 });
+  });
+
+  test('click collapse button', async ({ page }) => {
```

Comment:
Thanks for pointing this out will check into it

Reply:
Maybe something like this would help:

` await page.waitForChanges();`

I saw that was helping for other test where component has changed after interactions on component.